tests/fixtures/no-internal/workspace-pkg-1/estree.ts

```diff
@@ -2,4 +2,4 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-// test cases use this file
+// test cases use this file
```

Comment:
looks like there was no change here, should be reverted

Reply:
shouldn't be showing content diff and should only have "File renamed without changes." like the other ones. It might be line ending diff or something like that

Reply:
Fixed, the renamed file was missing a newline at the end